piqe_ocp_lib/api/resources/ocp_pods.py

```diff
@@ -183,3 +182,27 @@ def get_pod_node(self, namespace, pod_name):
             if api_response.spec["nodeName"]:
                 node_name = api_response.spec["nodeName"]
         return node_name
+    
+    def get_vm_pods_all_nodes(self):
+        """
+        Method that returns a list of All virtual_machine Pods belonging to
+        a Deployment Config in all namespaces
+        :return: dict of nodes containing list of virtual_machine
+
+        """
+        nodes_dict = {}
+        label_selector="kubevirt.io"
+        namespace="default"
+        api_response = self.ocp_pods.get(namespace=namespace, label_selector=label_selector)
+        if api_response:
```

Comment:
I'd like to propose some changes to this method as I feel it's doing more than it should in this scope. What I mean is the for the scope of a library, this method should just provide all pods associated with a node, and then maybe the part where we get the vm names from the pods can be handled at the pytest level. We are mixing native OCP/k8s resources with Operator crds. 
To be specific, this method should probably take a ```node_name``` as a required parameter and a ```label_selector``` as an optional parameter. Get all pods across namespaces (as ojbects) that match a label and/or are associated with a node and then return that. Then at the test script level we can retrieve the vm names. We can sync up and discuss in person if you'd like.

Reply:
done

Reply:
Have marked namespace and label_selector as optional params and resolved all linting issues.

Reply:
Do we need a namespace as an argument for this method? I might be wrong but I would think that we could have pods on the same node but be in different namespaces, in which cases we should just get all pods in all namespaces. There is already a method that does that in this class in line 94. I would suggest just modifying that method so it supports ```label_selector``` and then just call it here in line 196. The rest of the logic looks good. Let me know if you have anymore quiestion.

Reply:
Made the changes as per the discussion, marking this as resolved.